app-tracking-protection/vpn-impl/src/main/res/values/donottranslate.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright (c) 2023 DuckDuckGo
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+
+    <!-- Notify Me View on Downloads-->
+    <string name="appTrackingProtectionNotifyMeTitle">Keep an eye on blocked trackers</string>
```

Comment:
just to be clear, we don't need translations now but we are going to ask them later on? or simply we just won't do it since it's an experiment?

Reply:
The translations are not needed now. I'll do them separately from this release.

Reply:
Translations were added.